timur/docker/app/docker-entrypoint.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -e
+
+export PATH="/app/node_modules/.bin:/app/vendor/bundle/$RUBY_VERSION/bin:$PATH"
+cp npmrc ~/.npmrc || true
+
+if [ -z "$SKIP_RUBY_SETUP" ]; then
+  bundle check || bundle install -j "$(nproc)"
+  mkdir -p tmp/pids
+
+  rm -f tmp/pids/*.pid
+  if [ -z "$SKIP_DB_WAIT" ]; then
+    dockerize -wait tcp://timur_db:5432 -timeout 60s
+    ./bin/timur migrate
+  fi
+fi
+
+if [ -n "$RUN_NPM_INSTALL" ]; then
+  npm install --unsafe-perm
```

Comment:
Originally, to setup development links between the monorepo etna-js and metis, I had a make target `setup-links`.  However, I found out that `npm install` obliterates any `npm link` that you setup, so any container that would come up would incidentally destroy those links.

The fix i introduced into metis is to have, in this section here, a new section

```
  if [ -z "$MASTER_BUILD" ]; then
    echo 'Not master build, performing link to local etna-js...'
    npm link ../etna/packages/etna-js
  fi
```

The basic idea is that by default, containers actually do setup the `npm link` to the etna-js of the local checkout.  Only in master builds (in which we are create the special  etna-js artifact branch) do we not setup the npm link, so that it runs a final test against the package.json "as is" (since that is how it would run in production).  

If you decide to go this direction, too, to finish this setup, add a volume entry into the timur docker-compose for `- ../etna:/etna:cached` in addition to the above block here.